Developers/Unity Project/Spooksville/Assets/Scripts/AudioManager.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System;
+using UnityEngine;
+
+public class AudioManager : MonoBehaviour
+{
+    public static AudioManager instance;
+
+    public Sound[] sounds;
+
+    private void Start()
+    {
+        if (instance == null)
+        {
+            instance = this;
+            DontDestroyOnLoad(gameObject);
+        }
+    }
+
+    private void Awake()
+    {
+        foreach (Sound sound in sounds)
+        {
+            sound.source = gameObject.AddComponent<AudioSource>();
+            sound.source.clip = sound.clip;
+
+            sound.source.volume = sound.volume;
+            sound.source.pitch = sound.pitch;
+        }
+    }
+
+    public void Play(string name, bool isLooping)
```

Comment:
May want to consider adding a layer of indirection here to make it easier to rename/alias clips without having to edit all of the call sites. For instance you could have callers pass an enum instead of a string, which will also guard against typos.